exporter/awsemfexporter/grouped_metric.go

```diff
@@ -186,14 +192,28 @@ func mapGetHelper(labels map[string]string, key string) string {
 	return ""
 }
 
-func translateUnit(metric pmetric.Metric, descriptor map[string]MetricDescriptor) string {
+func translateUnit(metric pmetric.Metric, descriptor map[string]MetricDescriptor, value any) (string, any, error) {
 	unit := metric.Unit()
 	if descriptor, exists := descriptor[metric.Name()]; exists {
 		if unit == "" || descriptor.Overwrite {
-			return descriptor.Unit
+			return descriptor.Unit, value, nil
 		}
 	}
 	switch unit {
+	case "1":
+		unit = "Count"
+	case "ns":
```

Comment:
We used to do this in the agent, but recently changed it to a `None` unit (https://github.com/aws/amazon-cloudwatch-agent/pull/1322). The concern is that converting it to `Microseconds` lowers precision and might not be what the end user wants. This change would be especially strange for metrics that have `ns` in the name (there is one in the `jmx/kafka` metrics.

Reply:
Change it to `""`.